lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -193,15 +193,22 @@ def remove_copy_resources_script_phase_from_target(native_target)
           # @param [Class] phase_class
           #        The class of the phase to use.
           #
+          # @param [String] show_env_vars_in_log
+          #        Whether to show environment variables in the log during execution of this script phase.
+          #
           # @return [void]
           #
-          def create_or_update_build_phase(native_target, phase_name, phase_class = Xcodeproj::Project::Object::PBXShellScriptBuildPhase)
+          def create_or_update_build_phase(native_target, phase_name,
+                                           phase_class = Xcodeproj::Project::Object::PBXShellScriptBuildPhase,
+                                           show_env_vars_in_log = '0')
```

Comment:
this is only used for script phases, right? What happens when another phase class is passed in?

Reply:
actually all callers of this method are for script phases...we can even drop the `phase_class` parameter to be honest.

Reply:
even the documentation of the method says `# Creates or update a shell script build phase for the given target.`... I could even rename the method to make it explicit as well.

Reply:
@segiddins I went with the refactoring and its pushed.